spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryCommentsConverter.java

```diff
@@ -762,19 +831,12 @@ private void convertCommentFamily(FreeTextComment comment, UniProtDocument docum
                 .forEach(val -> updateFamily(val, document));
     }
 
-    private boolean hasExperimentalEvidence(List<Evidence> evidences) {
-        boolean experimental = false;
-        if (Utils.notNullNotEmpty(evidences)) {
-            experimental =
-                    evidences.stream()
-                            .map(Evidence::getEvidenceCode)
-                            .filter(Objects::nonNull)
-                            .anyMatch(code -> code == EvidenceCode.ECO_0000269);
-        }
-        return experimental;
-    }
-
-    private boolean hasExperimentalEvidence(Collection<String> evidences) {
-        return evidences.contains(EvidenceCode.ECO_0000269.name());
+    private boolean canAddExperimental(
```

Comment:
In general it would be better if we can see the possibility to simplify some methods in this class (long list of params/ huge method bodies etc)